### DISCUSSION TITLE
Can you use the java.util.Date type with the datePicker?

### DISCUSSION THREAD

**Discussion by private-main** [primefaces/community#2862] [PrimeFaces] [answered]
I noticed that Date is already deprecated. Will this type be removed for future use?

DatePicker <f:convertDateTime type="date" />
Will the above code become ineffective as well?

**Comment by jepsar** [comment]
This is an issue tracker. Please use https://github.com/orgs/primefaces/discussions/categories/primefaces for questions.

**Reply by melloware** [reply]
Moved here to the Discussions

**Reply by melloware** [reply]
`java.util.Date` while deprecated has been used in 20 years worth of java code and its not being removed from the JDK anytime soon or possibly ever.

**Reply by private-main** [reply]
Thank you for your response
I understand that the java.util.Date type may not be removed from the JDK. I want to ask if the value attribute of the datePicker component will ever lose support for the java.util.Date type.

**Reply by jepsar** [reply, marked as answer]
There are no plans to do that currently, and it's not likely such plans will be made anytime soon.
